pjsip-apps/src/pjsua/pjsua_app_legacy.c

```diff
@@ -1443,6 +1458,21 @@ static void ui_send_arbitrary_request()
     }
 }
 
+static void ui_toggle_call_sdp_offer()
+{
+    char input[32];
+
+    printf("Subsequent calls and UPDATEs will contain SDP offer [yes/no]:");
+    if (fgets(input, sizeof(input), stdin) == NULL)
```

Comment:
Sorry if it's not clear, but actually my suggestion was only to print out the message, so user can be informed.

Additional prompt should not be needed, since it creates an extra step that's not necessary.

Reply:
Now it prints out these messages:
```
You have 0 active call
>>> o
Subsequent calls and UPDATEs will contain SDP offer: YES.
>>> o
Subsequent calls and UPDATEs will contain SDP offer: NO.
```

Reply:
Hi @goodicus, please find the updated patch for cli.
[patch_cli2.patch](https://github.com/pjsip/pjproject/files/14630660/patch_cli2.patch)

Reply:
Hi @goodicus, please find the patch for the cli part.
[patch_cli2.patch](https://github.com/pjsip/pjproject/files/14630910/patch_cli2.patch)